src/parsers/standard.js

```diff
@@ -4,7 +4,7 @@ export default {
   parse (output) {
     let failedSpecs = new Set()
     let match = null
-    let FAILED_LINES = /at (?:\[object Object\]|Object)\.(?:<anonymous>|it) \((([A-Za-z]:\\)?.*?):.*\)/g
+    let FAILED_LINES = /at (?:\[object Object\]|Object|Context)\.(?:<anonymous>|it|beforeEach|afterEach|before|after) \((([A-Za-z]:\\)?.*?):.*\)/g
```

Comment:
This was already a bit unwieldy and is now pretty gnarly. What do you think about building this programatically? e.g. 

```javascript
const source = '\\[object Object\\]|Object|Context'
const contexts = '<anonymous>|it|beforeEach|afterEach|before|after'
const filepath = '(([A-Za-z]:\\\\)?.*?):.*'
const regex = `at (?:${source})\\.(?:${contexts}) \\(${filepath}\\)`

export default {
  parse (output) {
    let failedLines = new RegExp(regex, 'g')
    while (match = failedLines.exec(output)) { // eslint-disable-line no-cond-assign
```